test/test.py

```diff
@@ -228,122 +244,137 @@ def test_v5_bundle(self):
+        self._assert_bundle(bundle,
+                            diseases={'glioblastoma'})
 
     def test_vx_specimen_v271_bundle(self):
         """
         A bundle containing a specimen_from_organism.json with a schema version of 2.7.1
         """
-        self._test_bundle(uuid='70184761-70fc-4b80-8c48-f406a478d5ab',
-                          version='2018-09-05T182535.846470Z',
-                          deployment='staging',
-                          diseases={'glioblastoma'},
-                          library_construction_methods={'Smart-seq2'})
+        bundle = self._get_bundle(uuid='70184761-70fc-4b80-8c48-f406a478d5ab',
+                                  version='2018-09-05T182535.846470Z',
+                                  deployment='staging')
+        self._assert_bundle(bundle,
+                            diseases={'glioblastoma'},
+                            library_construction_methods={'Smart-seq2'})
 
     def test_preservation_storage_bundle(self):
         """
         A bundle with preservation and storage methods provided
         """
-        self._test_bundle(uuid='68bdc676-c442-4581-923e-319c1c2d9018',
-                          version='2018-10-07T130111.835234Z',
-                          deployment='staging',
-                          age_range=AgeRange(min=567648000.0, max=1892160000.0),
-                          diseases={'normal'},
-                          project_roles={'Human Cell Atlas wrangler', None, 'external curator'},
-                          storage_methods={'frozen, liquid nitrogen'},
-                          preservation_methods={'cryopreservation, other'},
-                          library_construction_methods={'Smart-seq2'},
-                          selected_cell_types={'TEMRA'},
-                          ncbi_taxon_ids={9606})
+        bundle = self._get_bundle(uuid='68bdc676-c442-4581-923e-319c1c2d9018',
+                                  version='2018-10-07T130111.835234Z',
+                                  deployment='staging')
+        self._assert_bundle(bundle,
+                            age_range=AgeRange(min=567648000.0, max=1892160000.0),
+                            diseases={'normal'},
+                            project_roles={'Human Cell Atlas wrangler', None, 'external curator'},
+                            storage_methods={'frozen, liquid nitrogen'},
+                            preservation_methods={'cryopreservation, other'},
+                            library_construction_methods={'Smart-seq2'},
+                            selected_cell_types={'TEMRA'},
+                            ncbi_taxon_ids={9606})
 
     def test_ontology_label_field(self):
         """
         A bundle in production containing a library_construction_approach field
         with "text" and "ontology_label" properties that have different values
         """
-        self._test_bundle(uuid='6b498499-c5b4-452f-9ff9-2318dbb86000',
```

Comment:
You shouldn't have to touch all this code. Why can't `_test_bundle` stay mostly the same return the bundle? Like so

```diff
Index: test/test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test/test.py	(revision 5e2b78954ac4fcd9efdd8547319c0ebaa7b0a733)
+++ test/test.py	(date 1604009199723)
@@ -354,15 +354,15 @@
         self.assertEqual(len(sequencing_protocols), 1)
         self.assertEqual(sequencing_protocols[0].paired_end, True)
 
-    def _test_bundle(self, uuid, version, replica='aws', deployment='prod', **assertion_kwargs):
+    def _test_bundle(self, uuid, version, replica='aws', deployment='prod', **assertion_kwargs) -> Bundle:
 
         manifest, metadata_files = self._load_bundle(uuid, version, replica=replica, deployment=deployment)
 
-        self._assert_bundle(uuid=uuid,
-                            version=version,
-                            manifest=manifest,
-                            metadata_files=metadata_files,
-                            **assertion_kwargs)
+        return self._assert_bundle(uuid=uuid,
+                                   version=version,
+                                   manifest=manifest,
+                                   metadata_files=metadata_files,
+                                   **assertion_kwargs)
 
     def _assert_bundle(self, uuid, version, manifest, metadata_files,
                        age_range=None,
@@ -379,7 +379,7 @@
                        is_sequencing_bundle=True,
                        slice_thickness=None,
                        ncbi_taxon_ids=None,
-                       content_description=None):
+                       content_description=None) -> Bundle:
         bundle = Bundle(uuid, version, manifest, metadata_files)
 
         # Every data file's manifest entry should be referenced by a metadata
@@ -493,6 +493,8 @@
             self.assertSetEqual(content_description,
                                 set(chain.from_iterable(file.content_description for file in bundle.files.values())))
 
+        return bundle
+
     dss_subscription_query = {
         "query": {
             "bool": {
```

Reply:
Ah. "...move the assertions to test_file_core_bundle" confused me, I thought you meant all the assertions.
I've removed the incorrect refactor commit and added a commit for the fixups.